input/docs/integrations/build-systems/index.cshtml

```diff
@@ -1,4 +1,5 @@
 Order: 90
+Description: Integration with build server
```

Comment:
```suggestion
Description: Integration with a build server
```

Reply:
@gep13 I understand your suggestion as "how to integrate with 1 specific build server", while my intention was to say something in the sense of "how to integrate with any build server" (so build server in plural than singular)

Reply:
`Integration with build servers` makes sense to me :+1: